about.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script src="script.js" defer></script>
+    <title>Estudio Polanco</title>
+  </head>
+  <body>
+    <div id="myNav" class="overlay">
+      <a href="javascript:void(0)" class="closebtn">&times;</a>
+      <div class="overlay-content">
+        <a href="#work" id="mybtn">Program</a>
```

Comment:
- [ ] Kindly fix the navigation to the home page from the about page!

Reply:
- [ ] As required by the previous TSE, kindly fix the navigation from the about page to the home page on mobile screens.
- The link to the home page doesn't show in the dropdown menu. 
- The menu items don't work, so navigation on mobile screens is impossible.
- The menu doesn't close after clicking on a menu item. Kindly fix this.

- [ ] Also, please follow the given design template, the fonts, spacings and colours.

Reply:
* @Richie9000 the issue here pointed out by the previous TSE has still not been addressed kindly make the necessary changes  🙏